examples/Azure_IoT_Hub_Arduino_Nano_RP2040_Connect/README.md

```diff
@@ -0,0 +1,289 @@
+
+## IoT Hub Setup
+1. In the Azure portal, navigate to your IoT Hub resource.
+
+1. On the left side menu under **Device management**, click on **'Devices'**.
+
+1. Select **'+ Add Device'**.
+
+1. Give your device a unique name.
+
+1. Select 'Symmetric key', 'Auto-Generate keys', and 'Enable' for Connecting the device to an IoT Hub, then click 'Save'.
+
+1. When the device has been created, select its name under 'Device ID'.
+
+1. We'll need the following information from the device page:
+
+    - Device ID
+    - Primary Key
+    - HostName (under Primary Connection String)
+
+_NOTE: Device keys are used to automatically generate a SAS token for authentication, which is only valid for one hour._
+
+## Arduino IDE Setup
+
+1. Open the Arduino IDE.
+
+1. Install Arduino Mbed OS Nano Boards support in the Arduino IDE. [Full instructions can be found here.](https://docs.arduino.cc/hardware/nano-rp2040-connect)
+
+    - Navigate to **Tools > Board > Board Manager**
+    - Search for 'RP2040' and install the **Arduino Mbed OS Nano Boards** core.
+    - Always install the latest version.    
+    
+    *Note: This process may take several minutes.*  
+
+1. Nagivate to **Tools > Board > Arduino Mbed OS Nano Boards** and select **'Arduino Nano RP2040 Connect'**.
+
+1. If this is your first time using the Nano RP2040 Connect, [follow these instructions to update the WiFi firmware on the Nano RP2040 Connect](https://docs.arduino.cc/tutorials/nano-rp2040-connect/rp2040-upgrading-nina-firmware).
+
+1. Install WiFiNINA library for the Nano RP2040 Embedded C SDK sample. 
+
+    - Navigate to **Tools > Manage Libraries**.
+    - Search for the **'WiFiNINA'** libray. Install the latest version.
+
+    *Note: This process may take several minutes.*  
+
+1. You may need to restart the Arduino IDE for changes to show up.
+
+
+## Run the Sample
+
+1. Open the Arduino Nano RP2040 Connect sample.
+
+    - In the Arduino IDE, navigate to **File > Examples > Azure SDK For C**
+    - Select **Azure_IoT_Hub_Arduino_Nano_RP2040_Connect** to open the sample. 
+
+1. Navigate to the '*iot_configs.h*' file
+
+1. In the *iot_configs.h* file, fill in your credentials. 
+
+    - Add in your WiFi SSID and password.
+    - Paste your IoT Hub device HostName for the *'IOT_CONFIG_IOTHUB_FQDN'* variable. It should look something like:
+
+        ```#define IOT_CONFIG_IOTHUB_FQDN "my-resource-group.azure-devices.net"```
+
+    - Paste your IoT Hub device ID for the *'IOT_CONFIG_DEVICE_ID'* variable.
+
+    - Finally, paste your IoT Hub Primary Key for the *'IOT_CONFIG_DEVICE_KEY'* variable.
+
+1. This sample was configured for a PST timezone (GMT -8hrs) with a Daylight Savings offset. If you live in a different timezone, update the values in '*Time Zone Offset*' at the bottom of the *iot_configs.h* file.
+
+    - Change the '*IOT_CONFIG_TIME_ZONE*' value to reflect the number of hours to add or subtract from the GMT timezone for your timezone.
```

Comment:
two comments here:
1. Is this really needed? no other IoT hub sample we have has this option... so just wondering if we're doing it wrong in the other samples...
2. even though you mention this option on the iot_configs file, I could not find this option there.